### PR TITLE
Add configuration for logrotate

### PR DESCRIPTION
**Pull request by foxdalas**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 maintainer_email 'jj@chef.io'
 license 'Apache-2.0'
 description 'Installs and configures RabbitMQ server'
-version '5.1.0'
+version '5.1.1'
```
Comment (michaelklishin) [inline metadata.rb]: Please revert this. It's up to the maintainers to decide what the next version should be (and why).

Comment (foxdalas) [inline metadata.rb]: fixed

**Comment by foxdalas** [comment]
fixed

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**